### DISCUSSION TITLE
Signup button

### DISCUSSION THREAD

**Discussion by Shashwat-pd** [monkeytypegame/monkeytype#3510] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/71143976/188483367-d96297f2-762f-4208-990b-97a71c7981e4.png)
The button on the extreme left was highly confusing. Although, I was not logged in it still felt like a logout button. Also, the two buttons navigated to the same place. It is a bit problematic in terms of UX. I suggest using only one button and making it more explicit by adding "Sign Up" text or an aria-label on hover.

**Comment by Miodec** [comment, marked as answer]
Ill hide the account button when signed out to avoid the ambiguity
